tests/test_dict.py

```diff
@@ -379,6 +379,30 @@ def test_save_reload(self):
             )
             assert da2.txt2vec("hello") == da.txt2vec("hello")
 
+    def test_add_special_tokens(self):
```

Comment:
would it be worth doing a test with a `transformer/generator`? Especially one that's had fp16?

Reply:
yeah i prob should